src/backend/model/user.model.js

```diff
@@ -0,0 +1,13 @@
+module.exports = class user {
```

Comment:
This type of model is the same model in Mongo projects, shouldn't use it to convert data form request body